geomesa-blobstore/geomesa-blobstore-api/src/main/scala/org/locationtech/geomesa/blob/api/BlobstoreServlet.scala

```diff
@@ -0,0 +1,51 @@
+package org.locationtech.geomesa.blob.api
+
+import java.io.{File, FileOutputStream}
+
+import com.google.common.io.Files
+import org.locationtech.geomesa.accumulo.data.{AccumuloDataStore, AccumuloDataStoreFactory}
+import org.locationtech.geomesa.blob.core.AccumuloBlobStore
+import org.locationtech.geomesa.web.core.GeoMesaScalatraServlet
+import org.scalatra.{NotFound, BadRequest, Ok}
+
+import scala.collection.JavaConversions._
+
+class BlobstoreServlet extends GeoMesaScalatraServlet {
+  override def root: String = "blob"
+
+  var abs: AccumuloBlobStore = null
+
+  post("/ds/:alias") {
```

Comment:
Hmm, I suggest getting access to the Geoserver catalog and crawling it for each of its stores and associated connection params.